### PR TITLE
[TC-362] Migrate MKR WiFi 1010 DS to MD

### PR DESCRIPTION
**Pull request by aliphys**
## What This PR Changes
- Migrate Google Docs version of MKR WiFi 1010 datasheet to MD

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jcarolinares** [comment]
@aliphys please try to run the Gatsby again, there was a technical problem that now should be already solved

**Comment by aliphys** [comment]
@jcarolinares Thanks for the edits.
Are there any other changes you would like to see or can we merge this PR?

**Comment by jcarolinares** [comment]
@aliphys yes, please solve this wrong link problem:

image

**Review by jcarolinares — CHANGES_REQUESTED** [review]

File: content/hardware/01.mkr/01.boards/mkr-wifi-1010/datasheet/datasheet.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+| Company Address | Via Andrea Appiani 25, 20900 MONZA, Italy                  |
+
+## Reference Documentation
+
+| Ref                       | Link                                                                                                |
+| ------------------------- | --------------------------------------------------------------------------------------------------- |
+| Arduino IDE (Desktop)     | https://www.arduino.cc/en/software                                                                  |
+| Arduino IDE (Cloud)       | https://create.arduino.cc/editor                                                                    |
+| Cloud IDE Getting Started | https://create.arduino.cc/projecthub/Arduino_Genuino/getting-started-with-arduino-web-editor-4b3e4a |
+| Arduino Documentation     | https://docs.arduino.cc/hardware/nano                                                               |
```
[the remark this answers is not in the document]
Comment (aliphys) [reply content/hardware/01.mkr/01.boards/mkr-wifi-1010/datasheet/datasheet.md:338]:
```suggestion
| Arduino Documentation     | https://docs.arduino.cc/hardware/mkr-wifi-1010                                                               |
```

**Pull request merged**